### DISCUSSION TITLE
Mobile platforms support

### DISCUSSION THREAD

**Discussion by yimengfan** [nxrighthere/UnrealCLR#8] [General]

**Comment by nxrighthere** [comment]
Only desktop platforms supported at the moment. We need .NET to support ahead-of-time compilation, please, see the [roadmap](https://github.com/users/nxrighthere/projects/5?fullscreen=true).

**Reply by nxrighthere** [reply]
@JimSimons-UH Please, see.

**Reply by leeveel** [reply]
why not use mono to support mobile platforms?

**Reply by nxrighthere** [reply]
@leeveel Mono doesn't support API required for this project, but it might change with .NET 7 and later versions.

**Reply by leeveel** [reply]
@nxrighthere could you be more detail, if there are other alternative ways? Thanks!

**Reply by nxrighthere** [reply]
> could you be more detail

Hosting, assembly management, framework API, and so on are different in Mono. It's in the unification phase with .NET Core since .NET 5 and it's not ready yet for usage in projects that are based on CoreCLR.

> if there are other alternative ways

There's an experimental [ahead-of-time toolchain branch](https://github.com/dotnet/runtimelab/tree/feature/NativeAOT) available   for a while, but it's a work in progress.

**Reply by lvzuoxin** [reply]
if NetCoreCLR support android and ios，this project can be usage？
